### DISCUSSION TITLE
GitLab / Hub API as Provider

### DISCUSSION THREAD

**Discussion by mirisbowring** [prowler-cloud/prowler#5430] [Ideas]
Hi,

are there any Plans to maybe also support the GitLab API to alter it and execute compliance checks against it?

Ideas:

* Are default branches protected
* Are multiple merge approvals configured
* Are e.g. Password Scans executed
* ...

**Comment by pedrooot** [comment]
Hi! @mirisbowring 
Adding GitLab or GitHub as a provider is an interesting proposal. We are currently evaluating with the team adding new providers, so we will take your request into consideration when making a decision.

Would you be interested in creating an issue to track your proposal? You can use the label feature-request.

**Reply by mirisbowring** [reply]
Created in #5446

**Reply by pedrooot** [reply]
Thanks!!
